include/asterisk/channel.h

```diff
@@ -606,6 +608,18 @@ struct ast_assigned_ids {
 	const char *uniqueid2;
 };
 
+/*!
+ * \brief Helper struct for initializing additional channel information on channel creation.
+ */
+struct ast_channel_initializers {
+	/*!
+	 * \brief struct ABI version
+	 * \note This \b must be incremented when the struct changes.
+	 */
+	#define AST_CHANNEL_INITIALIZERS_VERSION 1
```

Comment:
This doesn't place the value on the structure, so you can't actually do any checking or verification of the version of the underlying structured when using it. You can see an example with the CEL structure, it has a version field which gets set to it.